index.html

```diff
@@ -183,7 +251,267 @@ <h4 class="ring"><a href="#" class="over">Wolt</a></h4>
+<li><a href="#">Once you go EV, you never go back — according to 82% of owners, anyway</a></li>
+<li><a href="#">Waiters and employees are in charge — you're just paying them</a></li>
+<li><a href="#">GitOps has been 'controversial' — but the truth is your organization needs it</a></li>
+<li><a href="#">Tesla backtracks on hiring someone to fight Elon Musk’s Twitter critics</a></li>
+<li><a href="#">3 creative ways to use your brand’s data to improve marketing measures</a></li>
+</ul>
+</div>
+<div class="Launch">
+<div >
+<h4><a href="#" class="categories-title">Launch ></a> </h4>
+</div>
+<div class="categories-white7">
+<h5 class="popular1">MOST POPULAR</h5>
+<a href="#" class="game-stop">About time! Toyota finally announces an all-electric vehicle</a>
+<p class="categories-hour"><a href="#" class="categories-name">Matthew Beedham .</a>7 weeks ago</p>
+</div>
+<div class="categories-red7">
+<h5 class="latest-one">LATEST</h5>
+<a href="#" class="alibaba">I’ve fallen for the electric Hummer and I hate myself for it</a>
+</div>
+<ul class="categories-list">
+<li><a href="#">Amazon's electric delivery vans made by Rivian are here, and they're pretty cute</a></li>
+<li><a href="#">This adorable little electric car is made of trash</a></li>
+<li><a href="#">Hyundai's first fuel cell-powered trucks arrive in Switzerland — bye bye, diesel</a></li>
+<li><a href="#">Lynk & Co.'s car sharing-owning scheme will never cost more than €500 a month</a></li>
+<li><a href="#">Lucid unveils the Air, its luxury EV with up to 517-miles of range costing as much as $169K</a></li>
+</ul>
+</div>
+<div class="Distract">
+<div >
+<h4><a href="#" class="categories-title">Distract ></a> </h4>
+</div>
+<div class="categories-white8">
+<h5 class="popular1">MOST POPULAR</h5>
+<a href="#" class="game-stop">Amplitude celebrates 10 years with a free weekend of Endless games</a>
+<p class="categories-hour"><a href="#" class="categories-name">Tristan Greene .</a>4 days ago</p>
+</div>
+<div class="categories-red8">
+<h5 class="latest-one">LATEST</h5>
+<a href="#" class="alibaba">The best vampire games and mods to play until Bloodlines 2 releases later this year</a>
+</div>
+<ul class="categories-list">
+<li><a href="#">Games to play on date night: Rule the galaxy together in Stellaris</a></li>
+<li><a href="#">We need to make remote meetings more boring in 2021</a></li>
+<li><a href="#">The podcasts that got us through 2020</a></li>
+<li><a href="#">How Netflix shapes mainstream culture, explained by data</a></li>
+<li><a href="#">Christmas is better with Tom Vasel’s YouTube board game reviews</a></li>
+</ul>
+</div>
+</div>
+<div class="latest-deals">
+    <div class="latest-deal-top">
+        <h3>
+            <a href="#" class="top-deal-link">Latest deals <i class="fas fa-chevron-right"></i></a>
+        </h3>
+        <a href="" class="top-right-deal">See all deals</a>
+    </div>
+    <div class="latest-deals-container">
+        <div class="latest-deal-item">
```

Comment:
Kindly replace `<div class="latest-deal-item>` with `<article>` tag so we improve the use of HTML semantics. Do this for the others :smile: